### DISCUSSION TITLE
[Markdown] LaTeX multiple subscript bug

### DISCUSSION THREAD

**Discussion by JayWhite2357** [community/community#17384] [Projects and Issues]
When rendering subscripts on bold text, the LaTeX renderer fails. `${\bf{x}}_0, {\bf{x}}_{n-1}$` should look kind of like ${\bf{x_0}}, {\bf{x_{n-1}}}$ but instead renders as ${\bf{x}}_0, {\bf{x}}_{n-1}$.  

`${\bf{x}}_0$` seems to work sometimes: ${\bf{x}}_0$

`${\bf{x}}_0$` doesn't work other times: ${\bf{x}}_0$
`${\bf{x}}_{n-1}$` also works sometimes, but not here: ${\bf{x}}_{n-1}$  

`${\bf{x}}_{n-1}$` is working here: ${\bf{x}}_{n-1}$

**Comment by wkjung** [comment]
As a workaround, you can put backslash `\_` as suggested in https://github.com/github/feedback/discussions/16939.
`${\bf{x}}\_0, {\bf{x}}\_{n-1}$` => ${\bf{x}}\_0, {\bf{x}}\_{n-1}$

Seems like the renderer tries to interpret the underscore character as italics first.

**Reply by macdraw** [reply]
> As a workaround, you can put backslash `\_` as suggested in https://github.com/github/feedback/discussions/16939. `${\bf{x}}\_0, {\bf{x}}\_{n-1}$` => x0,xn−1
> 
> Seems like the renderer tries to interpret the underscore character as italics first.

**Comment by avm19** [comment]
This issue is known to the support. I included their response in the sister thread https://github.com/github-community/community/discussions/18897#discussion-4156463
